recipes/bornagain/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "BornAgain" %}
+{% set version = "1.16.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://apps.jcns.fz-juelich.de/src/BornAgain/BornAgain-{{ version }}.tar.gz
+  sha256: 20e390b683b3036aabf48d266e5e74da46ce9d781e72379dccb3d227ed300d40
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - boost
+    - fftw
+    - gsl
+    - qt 5.9.*
```

Comment:
We have Qt pinned globally at 5.12 https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/4f8f7f03140d73db3a0bf144ea537f466ca23fbf/recipe/conda_build_config.yaml#L568-569 Would that work here too?